### PR TITLE
#166789992: User Should Be Able To Signout

### PR DESCRIPTION
**Pull request by kodek-sleuth**
#### What does this PR do?
Signs out a logged in user
#### Description of Task to be completed
- Blacklist tokens on sign-out
- Setup `redis` an in-memory `db` that will store the blacklisted tokens 
#### How should this be manually tested?
1. Run the following commands in the terminal
```
git clone https://github.com/andela/ah-92explorers-backend.git
cd ah-92explorers-backend
git checkout ft-user-sign-out-166789992
npm install
npm start
```
2. Create a .env file and add the following snippets
```
DATABASE_URL=postgres://username:password@localhost:5432/authorshaven
DATABASE_URL=postgres://username:password@localhost:5432/testing-ah
```
3. Install `sequelize` globally
```
sudo npm install -g sequelize
```
4.  Create a database using `sequelize db:create`
5. Run the application with `npm start`
6. Use the following endpoints
## Endpoints
|  Method  |  Endpoint  |  Task  |
|  --- |  --- |  ---  |
|  `POST`  |  `/api/users/`  |  `signups a user`  |
|  `POST`  |  `/api/users/login`  |  `logs in a user`  |
|  `POST`  |  `/api/users/signout`  |  `signsout a logged in user`  |
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
[#166789992](https://www.pivotaltracker.com/story/show/166789992)
#### Screenshots (if appropriate)
Screen Shot 2019-07-11 at 10 58 11
#### Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I don't have more than 3 major commits on this PR

### CODE REVIEW

File: routes/api/admin.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import express from 'express';
+// import { checkToken, checkTokenExpiry } from '../../middlewares';
```
Comment (Hervera) [inline routes/api/admin.js]: Remove the comment

**Review by Hervera — APPROVED** [review]
Ready to be marged

**Review by IsaiahRn — APPROVED** [review]
After, testing it locally in my computer, this PR is ready to be merged!

**Review by elemanhillary-zz — APPROVED** [review]
ready for merge

**Pull request merged**